.github/workflows/ci-workflow.yml

```
@@ -75,6 +75,16 @@ jobs:
       - name: Install dependencies
         run: yarn install
 
+      - name: Build project
+        run: yarn build
+  
+      - name: Upload source maps to Sentry
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+          SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
+        run: npx sentry-cli sourcemaps upload ./packages/extension/build --org $SENTRY_ORG --project $SENTRY_PROJECT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:67: Double quote to prevent globbing and word splitting

(shellcheck)